serf/merge_delegate.go

```diff
@@ -15,24 +17,33 @@ type mergeDelegate struct {
 }
 
 func (m *mergeDelegate) NotifyMerge(nodes []*memberlist.Node) error {
+	var err error
```

Comment:
tiny nitpick, but this could go in the for loop since it's only being used in that scope